torch/quantization/observer.py

```diff
@@ -171,40 +197,87 @@ def _calculate_qparams(self, min_val, max_val):
 
 
 class MinMaxObserver(_ObserverBase):
-    r"""Default Observer Module
-    A default implementation of the observer module, only works for
-    `per_tensor_affine` quantization scheme.  The module will record the
-    running average of max and min value of the observed Tensor and
-    calculate_qparams will calculate scale and zero_point
+    r"""Observer module for computing the quantization parameters based on the
+    running min and max values.
+
+    This observer uses the tensor min/max statistics to compute the quantization
+    parameters. The module records the running minimum and maximum of incoming
+    tensors, and uses this statistic to compute the quantization parameters.
+
+    Args:
+        dtype: Quantized data type
+        qscheme: Quantization scheme to be used
+        reduce_range: Reduces the range of the quantized data type by 1 bit
+
+    Given running min/max as :math:`x_\text{min}` and :math:`x_\text{max}`,
+    scale :math:`s` and zero point :math:`z` are computed as:
+
+    The running minimum/maximum :math:`x_\text{min/max}` is computed as:
+
+    .. math::
+
+        x_\text{min/max} = \begin{cases}
+            \frac{\min}{\max}(X) & \text{if~}x_\text{min/max} = \O \\
+            \frac{\min}{\max}\left(x_\text{min/max},
+            \frac{\min}{\max}(X)\right) & \text{otherwise}
+        \end{cases}
```

Comment:
![image](https://user-images.githubusercontent.com/4216323/66863862-c5feba80-ef48-11e9-9980-1b772e0cedc2.png)

Reply:
Lets have separate equations for min and max, so that it is clearer